src/Sylius/Behat/spec/Context/Transform/AddressingContextSpec.php

```diff
@@ -57,4 +60,16 @@ function it_creates_new_address_based_on_given_country(
 
         $this->createNewAddress('France')->shouldReturn($address);
     }
+
+    function it_gets_country_based_on_given_country_name(
+        CountryNameConverterInterface $countryNameConverter,
+        RepositoryInterface $countryRepository,
+        CountryInterface $country
+    ) {
+        $countryNameConverter->convertToCode('France')->willReturn('FR');
+        $countryRepository->findOneBy(['code' => 'FR'])->willReturn($country);
+        $country->getCode()->willReturn('FR');
+
+        $this->getCountryByName('France')->shouldReturn($country);
+    }
```

Comment:
Missing two specs: when country is not found and invalid country name is passed (convertToCode will throw exception).